components/nexus-core/src/main/java/org/sonatype/nexus/internal/app/BaseUrlManagerImpl.java

```diff
@@ -40,15 +40,18 @@
 {
   private final Provider<HttpServletRequest> requestProvider;
 
-  @Inject
-  public BaseUrlManagerImpl(final Provider<HttpServletRequest> requestProvider) {
-    this.requestProvider = checkNotNull(requestProvider);
-  }
-
   private volatile String url;
 
   private volatile boolean force;
 
+  @Inject
+  public BaseUrlManagerImpl(final Provider<HttpServletRequest> requestProvider,
+                            @Named("${org.sonatype.nexus.internal.app.BaseUrlManagerImpl.force:-false}") final boolean force) {
```

Comment:
I never plan to tell anyone to use this property - so I guess I don't care too much that it doesn't follow the convention of nexus.feature.subfeature